docs/griptape-framework/drivers/event-listener-drivers.md

```diff
@@ -0,0 +1,53 @@
+## Overview
+
+Event Listener Drivers are used to send Griptape [Events](../misc/events.md) to external services.
```

Comment:
not to get pedantic, but is Skatepark emu an "external service"

Reply:
I'd say so, yeah. I'd consider anything that is not part of the running Griptape program an external service.

Reply:
I'll yield here